.github/config.yml

```diff
@@ -0,0 +1,46 @@
+---
+#-------------------------------------------------------------------------------
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening this issue. A contributor will be by to give feedback soon. In the meantime, please review the [Layer5 Contributors' Welcome Guide](https://docs.google.com/document/d/17OPtDE_rdnPQxmk2Kauhm3GwXF1R5dZ3Cj8qZLKdo5E/edit?usp=sharing), engage in the [discussion forum](https://discuss.layer5.io), and be sure to join the [community Slack](http://slack.layer5.io/).
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Yay, your first pull request! :thumbsup: A contributor will be by to give feedback soon. In the meantime, you can find updates in the [#github-notifications](https://layer5io.slack.com/archives/CLVT4TSG4) channel in the [community Slack](https://slack.layer5.io).
+
+  Be sure to double-check that you have signed your commits. Here are instructions for [making signing an implicit activity while peforming a commit](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
+
+#-------------------------------------------------------------------------------
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Thanks for your contribution to Meshery! :tada:
+   
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset=".github/welcome/Meshery_Welcome_light.png">
+  <source media="(prefers-color-scheme: light)" srcset=".github/welcome/Meshery_Welcome_light.png">
+  <img alt="Meshery Logo" src="">
+  </picture>
+
```

Comment:
Here I think @shashanksjSSJ , `srcset` needs to be a reference to absolute path to raw image.
Also the in the `srcset` both the images you have named as Meshery_Welcome_light so correct that too one should be named dark.
I'm also mentioning a PR where you can get better idea of what i'm talking about - [Meshkit - #308](https://github.com/meshery/meshkit/pull/308)

Reply:
@shashanksjSSJ for your contribution. 
Can you please use SVG format for the images and keep the files name in lowercase besides having an absolute path?

Reply:
I need to change the raw link to /meshery/meshery right